### PR TITLE
Release 1.0.1

### PR DESCRIPTION
**Pull request by chadell**

### CODE REVIEW

File: changes/259.added [hunk 1]
```diff
@@ -0,0 +1 @@
+Added screenshots to docs/images.
```
Comment (glennmatthews) [inline changes/259.added:1]: Should these files be removed now that they're reflected in `docs/release_notes/version_1.0.md`? Normally I'd expect `towncrier` to do that for us.

File: docs/dev/dev_environment.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Using **Invoke** these configuration options can be overridden using [several me
 ### Docker Development Environment
 
 !!! tip
-    This is the recommended option for development.
+This is the recommended option for development.
```
Comment (glennmatthews) [inline docs/dev/dev_environment.md:28]: This looks wrong - it should be reverted.

File: docs/dev/dev_environment.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ nautobot-server migrate
 ```
 
 !!! note
-    If you want to develop on the latest develop branch of Nautobot, run the following command: `poetry add --optional git+https://github.com/nautobot/nautobot@develop`. After the `@` symbol must match either a branch or a tag.
+If you want to develop on the latest develop branch of Nautobot, run the following command: `poetry add --optional git+https://github.com/nautobot/nautobot@develop`. After the `@` symbol must match either a branch or a tag.
```
Comment (glennmatthews) [inline docs/dev/dev_environment.md:76]: This looks wrong - it should be reverted.

File: docs/dev/dev_environment.md [hunk 4]
```diff
@@ -136,15 +133,14 @@ Each command can be executed with `invoke <command>`. All commands support the a
...
 
 The upside to having the Nautobot service handled by Docker rather than locally is that you do not have to manage the Nautobot server. The [Docker logs](#docker-logs) provide the majority of the information you will need to help troubleshoot, while getting started quickly and not requiring you to perform several manual steps and remembering to have the Nautobot server running in a separate terminal while you develop.
 
 !!! note
-	The local environment still uses Docker containers for the supporting services (Postgres, Redis, and RQ Worker), but the Nautobot server is handled locally by you, the developer.
+The local environment still uses Docker containers for the supporting services (Postgres, Redis, and RQ Worker), but the Nautobot server is handled locally by you, the developer.
```
Comment (glennmatthews) [inline docs/dev/dev_environment.md:143]: Both the before and after appear wrong here - there should be a four-space indent to this paragraph. Same comment on all other instances in this file below.

File: pyproject.toml [hunk 5]
```diff
@@ -18,51 +16,47 @@ packages = [
...
 
 [tool.poetry.dev-dependencies]
-invoke = "*"
-# Black 23.x.x configuration changes and migration files are taken into account
-black = "^22.6.0"
-django-debug-toolbar = "*"
-yamllint = "*"
+Markdown = "*"
 bandit = "*"
+black = "*"
```
Comment (glennmatthews) [inline pyproject.toml:33]: Changing `black` from a pinned dependency `^22.6.0` to an unpinned `*` dependency seems odd to me. Should we instead be bumping it to a newer pinned version?

File: changes/259.added (hunk 1 above)
Comment (chadell) [reply changes/259.added:1]: towncrier, in this repository, it's only checking for this file per PR, but it's not doing anything for the release notes (I may be missing something)

File: pyproject.toml (hunk 5 above)
Comment (chadell) [reply pyproject.toml:33]: this was reverted to the cookiecutter standards (initial one). I could pin a specific one if required

**Review by pke11y — APPROVED** [review]
LGTM if you can resolve Glenn's comments.

**Pull request merged**